README.md

```diff
@@ -13,44 +13,115 @@ View the [list of supported APIs and Services](https://cloud.google.com/php/docs
 
 If you need support for other Google APIs, please check out the [Google APIs Client Library for PHP](https://github.com/google/google-api-php-client).
 
-## Quick Start
+## Quickstart
 
-We recommend installing individual component packages. A list of available packages can be found on [Packagist](https://packagist.org/search/?q=google%2Fcloud-).
+In this guide we'll focus on how to configure your client for local development using the Google
+Cloud CLI (`gcloud`).
 
-For example:
+### For local development:
+* Install the Google Cloud CLI.
+* Authenticate with `gcloud` to generate the credentials file.
+* Instantiate a client.
 
-```sh
-$ composer require google/cloud-storage
-$ composer require google/cloud-bigquery
-$ composer require google/cloud-datastore
+### Installing the Google Cloud CLI
+In order to generate our needed credentials file we need to authenticate to gcloud first.
+Installation is handled differently depending on your platform. Here is a link to help you setup
+the Google Cloud CLI:
+
+https://cloud.google.com/sdk/docs/install
+
+### Authenticate via the `gcloud` command
+Once the Google Cloud CLI tools are installed it is required that we authenticate via the `gcloud`:
+```shell
+$ gcloud init
+$ gcloud auth application-default login
+```
+
+This will create a local file in your system that the authentication library for our client will
+read in order to make requests to the apis with those credentials. This file is located in different
+place depending on your system.
+
+Windows:
+```
+%APPDATA%\gcloud\application_default_credentials.json
+```
+
+Linux and MacOS:
+```
+$HOME/.config/gcloud/application_default_credentials.json
 ```
 
-You can then include the autoloader and create your client:
+To read more about Authentication, see [AUTHENTICATION.md](AUTHENTICATION.md)
+
+### Instantiating the client
+Once we have the `application_default_credentials.json` that we created on the previous step now we
+can instantiate a client which internally using the Google Auth library will take that file and use
+it to authenticate your requests:
+
+**Note**: You can install the Google Translate client library by running `composer install google/cloud-translate`.
```

Comment:
Since this a quickstart, let's make this a required step (rather than a **NOTE**):

````suggestion
Install the Google Translate client library with composer

```sh
composer install google/cloud-translate
```
````